README.md

```diff
@@ -12,6 +12,71 @@ transit_model is a rust crate to manage transit data. Its model corresponds to t
 cargo build --release
 ```
 
+### Compile for KV1
+The KV1 format needs a additional dependency called [Proj.4](https://proj4.org/)
+which allows the transformation of localization coordinates.
+[crates.io](https://crates.io/) provides a
+[`proj`](https://crates.io/crates/proj) crate which is a binding to the C
+library (version 6.0.0). This means you need this version installed on your
+system. The commands below will build and install [Proj.4](https://proj4.org/) on your system.
+
+```
+apt install \
```

Comment:
I'd be OK with that if the documentation of Proj.4 was sufficient. But they don't explicitly mention the needed dependencies and since it took some time to figure them out, I think documenting them is useful. I can do a PR to the Proj.4 project if you think it's a better idea.

Reply:
the installation is done in the Dockerfile, you can suggest to see the Dockerfile

Reply:
Good idea, I'll do that (and I still might get a PR for Proj.4 project too).